tfhe/src/lib.rs

```diff
@@ -2,6 +2,9 @@
 //!
 //! TFHE-rs is a fully homomorphic encryption (FHE) library that implements Zama's variant of TFHE.
 
+// Enable all warnings in doctests
+// https://doc.rust-lang.org/rustdoc/write-documentation/documentation-tests.html#showing-warnings-in-doctests
+#![doc(test(attr(warn(unused))))]
```

Comment:
so as I said somewhere else, let's not have that in the user facing docs, otherwise it just looks ridiculous, I'm not convinced for doctests in general either, it looks very bad to have to essentially to prefix everything with _, but I'm guessing we are missing the unused imports if we allow that in that case ?